### PR TITLE
ticket 5 complete with error handling and tests

### PR DESCRIPTION
**Pull request by Pink-Flag**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -77,3 +77,74 @@ describe("tests for get article by ID", () => {
...
+          created_at: "2020-07-09T20:11:00.000Z",
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          votes: -10,
+        });
+      });
+  });
+  it("status 404: returns a not found message when article ID isn't in db", () => {
+    return request(app)
+      .patch("/api/articles/9999999")
```
Comment (dappernerddesigns) [inline __tests__/app.test.js:144]: There is no send here to fully test this error

File: __tests__/app.test.js [hunk 2]
```diff
@@ -77,3 +77,74 @@ describe("tests for get article by ID", () => {
...
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("not found");
+      });
+  });
```
Comment (dappernerddesigns) [inline __tests__/app.test.js:149]: What happens if a vote object is sent with no numbers, or an invalid data type? Please add these tests. I know your front end should stop this happening, but don't forget you can interact with apis through tools like insomnia and break them!

**Review by dappernerddesigns — CHANGES_REQUESTED** [review]
Please add more tests to your patch to fully check the functionality :) DM me when done.

File: __tests__/app.test.js (hunk 2 above)
Comment (Pink-Flag) [reply __tests__/app.test.js:149]: of course! thanks!

File: __tests__/app.test.js (hunk 1 above)
Comment (Pink-Flag) [reply __tests__/app.test.js:144]: ah sorry!

**Review by Pink-Flag — COMMENTED** [review]
added extra error handling

**Review by dappernerddesigns — APPROVED** [review]
Great stuff, just before the lunch cut off ;) merge when ready

**Pull request merged**